<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_reservaciones</class>
 <widget class="QDialog" name="admin_reservaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>651</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>220</y>
     <width>241</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>79</y>
     <width>231</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="SideBarLetras">
    <item>
     <widget class="QLabel" name="elPatio_4">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #585858
</string>
      </property>
      <property name="text">
       <string>Informaci√≥n de Reserva</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="elPatio_6">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #585858</string>
      </property>
      <property name="text">
       <string>Fecha de Reserva</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fechaReserva">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>24/Enero/2020</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="elPatio_11">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #585858</string>
        </property>
        <property name="text">
         <string>Hora de llegada:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="elPatio_7">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #585858</string>
        </property>
        <property name="text">
         <string>Mesa:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBoxMesas">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="elPatio_8">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #585858</string>
      </property>
      <property name="text">
       <string>Num. Personas</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnMenosExtra">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/iconos/resources/menosPlatillo.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numPersonas">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#858585;
</string>
        </property>
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMasExtra">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/iconos/resources/masPlatillo.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="elPatio_9">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #585858</string>
      </property>
      <property name="text">
       <string>Nombre a Reservar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_nombrePlatillo">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;

border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Ej. Benito Camelo</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_total">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>700</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="MenuTop">
    <item>
     <layout class="QVBoxLayout" name="LetrasMenu">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_2">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #585858
</string>
        </property>
        <property name="text">
         <string>Reservaciones</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elPatio_3">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Agenda nuevas reservas</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="BtnsMenu">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCambiarMesero">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
        </property>
        <property name="text">
         <string>Transferir cuenta</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>731</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Encabezados2">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="elPatio">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>El Patio</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1021</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrearReserva">
   <property name="geometry">
    <rect>
     <x>759</x>
     <y>700</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
   </property>
   <property name="text">
    <string>Crear Reserva</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Encabezados">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="nombreMesero">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>Fecha Hoy</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="sideBar">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>300</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 247, 244);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>162</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="elPatio_10">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #585858</string>
      </property>
      <property name="text">
       <string>Mes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cBoxMes">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
border-radius: 5px;</string>
      </property>
      <item>
       <property name="text">
        <string>Enero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Febero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Marzo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Abril</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mayo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Junio</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Julio</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agosto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Septiembre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Octubre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Noviembre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diciembre</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
    </font>
   </property>
   <property name="text">
    <string>Aqui va el calendario o eso xd</string>
   </property>
  </widget>
  <zorder>sideBar</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>lb_total</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>line</zorder>
  <zorder>btnCrearReserva</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>elPatio_11</zorder>
  <zorder>timeEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
